sherpa/astro/utils/xspec.py

```diff
@@ -876,8 +876,7 @@ def simple_wrap(modelname: str, mdl: ModelDefinition) -> str:
     # initialized.
     #
     if nflags > 1 and mdl.flags[1] == 1:
-        out += f"{t2}self._use_caching = False\n"
-
+        out += f"{t2}self.cache = 0\n"
```

Comment:
I'm surprised there's no test that needs to be changed because of this. Maybe I just never included a test of this combination.